### PR TITLE
Change icons throughout sciNote - navigation, project, task cards [SCI-2271]

### PR DESCRIPTION
**Pull request by mlorb**
[SCI-2271](https://biosistemika.atlassian.net/browse/SCI-2271)

Check:
[billing](https://gitlab.com/biosistemika/scinote/addons/billing/merge_requests/42)
[electronic_signatures](https://gitlab.com/biosistemika/scinote/addons/electronic_signatures/merge_requests/20)
[gamification](https://gitlab.com/biosistemika/scinote/addons/gamification/merge_requests/18)
[organization_management](https://gitlab.com/biosistemika/scinote/addons/organization_management/merge_requests/20)

### CODE REVIEW

File: app/assets/javascripts/repositories/repository_datatable.js.erb [hunk 1]
```diff
@@ -1165,14 +1165,14 @@ var RepositoryDatatable = (function(global) {
       var column = TABLE.column(li.attr('data-position'));
 
       if (column.visible()) {
-        self.addClass('glyphicon-eye-close');
-        self.removeClass('glyphicon-eye-open');
+        self.addClass('fa-eye-slash');
```
Comment (ZmagoD) [inline app/assets/javascripts/repositories/repository_datatable.js.erb]: do we also need to add .fas class beside that or no?

File: app/assets/stylesheets/projects.scss [hunk 2]
```diff
@@ -417,7 +417,7 @@ li.module-hover {
       }
     }
 
-    .glyphicon {
+    .glyphicon, .fas {
```
[the remark this answers is not in the document]
Comment (ZmagoD) [reply app/assets/stylesheets/projects.scss]: do we still use glyphicon-s?

[the remark this answers is not in the document]
Comment (mlorb) [reply app/assets/stylesheets/projects.scss]: Yes, most of the app still uses glyphicon-s.

File: app/assets/javascripts/repositories/repository_datatable.js.erb (hunk 1 above)
Comment (mlorb) [reply app/assets/javascripts/repositories/repository_datatable.js.erb]: No, fas is defined in the html.

**Pull request merged**